src/modules/config/pipe/browser-config-name.pipe.ts

```diff
@@ -0,0 +1,24 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+import {Observable} from 'rxjs';
+import {first, map} from 'rxjs/operators';
+import {ConfigObject} from '../../../shared/models/config';
+
+@Pipe({
+  name: 'getBrowserConfigName'
+})
+export class BrowserConfigNamePipe implements PipeTransform {
+  constructor(private route: ActivatedRoute) {
+  }
+
+  transform(configObject: ConfigObject): Observable<string> {
+    console.log('browserconfigName: ', configObject);
```

Comment:
Please remove console.log statement

Reply:
console.log statement removed in 993a64c4494ecb0516dbefee573d7ca875a60100